includes/admin/class-wcmp-export.php

```diff
@@ -159,6 +177,7 @@ public function admin_notices()
      * Export selected orders.
      *
      * @access public
+     * @var $order_ids
```

Comment:
Zet var in de code zelf, boven waar hij gebruikt wordt. Voeg er ook meteen een type aan toe anders heb je er nog niks aan